src/makeRobotAccountant.js

```diff
@@ -6,7 +6,37 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let callCount = 0;
+  let resetCallCount = 0;
+
+  const getSum = (a) => {
+    return (b) => {
+      if (callCount === 3) {
+        callCount++;
+        resetCallCount++;
+
+        return 'Bzzz... Error!';
+      };
+
+      if (callCount > 3 && resetCallCount % 2 !== 0) {
+        resetCallCount++;
+
+        return a + b;
+      };
+
+      if (callCount > 3 && resetCallCount % 2 === 0) {
+        resetCallCount++;
+
+        return 'Bzzz... Error!';
+      }
```

Comment:
No need to create 3 if
```suggestion
      if (callCount > 3 && callCount % 2 === 0) {
        count++;

        return 'Bzzz... Error!';
      }
      count++;

      return a + b;
```

Reply:
it does not work as the task requires. The error appears only on the 4th step, not on the 3rd.
![image](https://github.com/mate-academy/js_sum-with-closure/assets/118197236/33b0635c-5289-4658-a553-1e661873f7ed)

Reply:
@IvanFesenko